pkg/util/namer/namer_test.go

```diff
@@ -77,6 +76,19 @@ func TestGetNameIsDifferent(t *testing.T) {
 	}
 }
 
+func TestGetNameReturnShortNames(t *testing.T) {
+	base := randSeq(32)
+	for maxLength := 0; maxLength < len(base)+2; maxLength++ {
+		for suffixLen := 0; suffixLen <= maxLength+1; suffixLen++ {
+			suffix := randSeq(suffixLen)
+			got := GetName(base, suffix, maxLength)
+			if len(got) > maxLength {
+				t.Fatalf("len(GetName(%[1]q, %[2]q, %[3]d)) = len(%[4]q) = %[5]d; want %[3]d", base, suffix, maxLength, got, len(got))
```

Comment:
I used `t.Fatalf` here on purpose, as there's no benefit to see hundreds of lines with random strings in the output.